### PR TITLE
fix: fixed build error #14

### PR DESCRIPTION
**Pull request by ChenYCL**

### CODE REVIEW

File: apps/navbar/package.json [hunk 1]
```diff
@@ -4,18 +4,19 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node server.js",
-    "dev": "NODE_ENV=development nodemon server.js",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "npm run build:client && npm run build:server",
-    "build:client": "webpack --config webpack.config.js --progress --hide-modules",
-    "build:server": "webpack --config webpack.server.js --progress --hide-modules"
+    "start": "cross-env node server.js",
```
Comment (StyleT) [inline apps/navbar/package.json:7]: Hi!
Thank you for your contribution :) 
However I think you have a mistake here. `cross-env` should be added only to those scripts which contain env vars. 
So this is correct: `cross-env NODE_ENV=development nodemon server.js` but this doesn't `cross-env node server.js`

Comment (ChenYCL) [reply apps/navbar/package.json:7]: hi, I modified again. plz check .

Comment (StyleT) [reply apps/navbar/package.json]: Hi! Could you pls do the same in other `package.json` files? And don't forget to remove `"cross-env": "^7.0.2",` for packages where you won't use it eventually.

Comment (ChenYCL) [reply apps/navbar/package.json]: ok , i will check it and remove the package if don't use it .

**Comment by StyleT** [comment]
Thank you for the contribution 🤘 🔥

**Pull request merged**